website/src/components/Devices.tsx

```diff
@@ -35,7 +35,7 @@ export const Devices = observer(
     }
 
     componentWillUnmount() {
-      this.devices.dispose();
```

Comment:
maybe some can have a look at this. The Dispose makes a error because of the strict mode:

With Strict Mode in React 18, React will simulate unmounting and remounting the component in development mode

the remounting is here the problem